docs/DeveloperGuide.md

```diff
@@ -997,6 +1089,25 @@ testers are expected to do more *exploratory* testing.
 
 ---
 
+### Suggesting a show
+
+1. Prerequisites: None.
+2. Launch the app.
+3. Command: `suggest`
+  1. Test case:
+    1. Condition: Displayed list of show is empty.
+    2. Expected: Trackermon returns an error message informing user that displayed list of show is empty.
+  2. Test case:
+    1. Condition: Displayed list of show only contains 1 show.
+    2. Expected: Trackermon returns an error message informing user that there is only 1 show in the displayed list of show.
+  3. Test case:
+    1. Condition: Displayed list of show with 2 or more show.
```

Comment:
Likewise, two instead of 2?
Also, should it be `Displayed list of shows contains two or more shows`